pkg/manager/member/pvc_resizer.go

```diff
@@ -395,51 +404,75 @@ func (p *pvcResizer) resizeVolumes(ctx *componentVolumeContext) error {
 	desiredVolumeQuantity := ctx.desiredVolumeQuantity
 	podVolumes := ctx.actualPodVolumes
 
+	tcID := fmt.Sprintf("%s/%s", ctx.namespace, ctx.name)
+
 	for _, podVolume := range podVolumes {
+
+		allResized := true
+
 		for volName, pvc := range podVolume.volToPVCs {
+			pvcID := fmt.Sprintf("%s/%s", pvc.Namespace, pvc.Name)
+
+			// check whether the PVC is resized
 			quantityInSpec, exist := desiredVolumeQuantity[volName]
 			if !exist {
-				klog.Warningf("PVC %s/%s does not exist in desired volumes", pvc.Namespace, pvc.Name)
+				klog.Errorf("Check PVC %q of TC %q resized failed: not exist in desired volumes", pvcID, tcID)
 				continue
 			}
-
-			// not support default storage class
-			if pvc.Spec.StorageClassName == nil {
-				klog.Warningf("PVC %s/%s has no storage class, skipped", pvc.Namespace, pvc.Name)
+			currentRequest, ok := pvc.Spec.Resources.Requests[corev1.ResourceStorage]
+			if !ok {
+				klog.Errorf("Check PVC %q of TC %q resized failed: storage request is empty", pvcID, tcID)
 				continue
 			}
-
-			// check whether the volume needs to be expanded
-			currentRequest, ok := pvc.Spec.Resources.Requests[corev1.ResourceStorage]
+			currentCapacity, ok := pvc.Status.Capacity[corev1.ResourceStorage]
 			if !ok {
-				klog.Warningf("PVC %s/%s storage request is empty, skipped", pvc.Namespace, pvc.Name)
+				klog.Errorf("Check PVC %q of TC %q resized failed: storage capacity is empty", pvcID, tcID)
 				continue
 			}
+
 			cmpVal := quantityInSpec.Cmp(currentRequest)
+			resizing := currentRequest.Cmp(currentCapacity) != 0
 			if cmpVal == 0 {
-				klog.V(4).Infof("PVC %s/%s storage request is already %s, skipped", pvc.Namespace, pvc.Name, quantityInSpec.String())
+				if resizing {
+					allResized = false
+					klog.Infof("PVC %q of TC %q is resizing, request: %s capcaity: %s",
+						pvcID, tcID, currentRequest.String(), currentCapacity.String())
+				} else {
+					klog.V(4).Infof("PVC %q of TC %q is already resized, request: %s",
+						pvcID, tcID, quantityInSpec.String())
+				}
 				continue
 			}
+
+			allResized = false
```

Comment:
Why set to false here? no update to the PVC if `cmpVal < 0`, right?

Reply:
It means the PVC need to resize, so `allResize` is set to false and don't resize other PVCs of other Pods

Reply:
moved it after checking, PTAL